src/components/ui/Tiles/Tiles.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { forwardRef } from 'react'
+import type { HTMLAttributes, ReactElement } from 'react'
+
+import Tile from './Tile'
+
+export interface TilesProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+}
+
+const MIN_CHILDREN = 2
+const MAX_CHILDREN = 4
+const NUMBER_ITEMS_TO_EXPAND = 3
+
+const Tiles = forwardRef<HTMLDivElement, TilesProps>(function Tiles(
+  { testId = 'store-tiles', children, ...otherProps },
+  ref
+) {
+  const childrenCount = React.Children.count(children)
+  const spanFirstChild = childrenCount === NUMBER_ITEMS_TO_EXPAND
+  const isOutOfBounds =
+    childrenCount < MIN_CHILDREN || childrenCount > MAX_CHILDREN
+
+  if (isOutOfBounds) {
+    throw new Error(
+      `Tiles cannot receive less than ${MIN_CHILDREN} or more than ${MAX_CHILDREN} children.`
+    )
+  }
```

Comment:
Does this error is being used to warn the developer about the bounds? What do you think about only warning on developing mode?

```suggestion
  if (process.env.NODE_ENV === 'development') {
    if (isOutOfBounds) {
      throw new Error(
        `Tiles cannot receive less than ${MIN_CHILDREN} or more than ${MAX_CHILDREN} children.`
      )
    }
  }
```

Reply:
We'd like to ensure at this moment that we just have from 2 to 4 children as `Tile`. I think the [same comment above](https://github.com/vtex-sites/base.store/pull/141#discussion_r772730022) applies here. What do you think?